winlogbeat/eventlog/wineventlog.go

```diff
@@ -448,14 +447,6 @@ func (l *winEventLog) Read() ([]Record, error) {
 	for _, h := range handles {
 		l.outputBuf.Reset()
```

Comment:
> Anyway unused memory from the buffer will just get paged out at runtime.

It's not just a memory cost.  This `Reset()` call will now have to zero 512KB instead of 8KB, even for small messages.  We're optimizing for the large-message case at the expense of the small-message one.